config/development.js

```diff
@@ -0,0 +1,23 @@
+// Config specific to local development
+
+const defer = require('config/defer').deferConfig;
+
+module.exports = {
+  serverPort: 3000,
+
+  apiHost: 'https://addons-dev.allizom.org',
```

Comment:
Oh that needs to be done differently to use the proper node-config way. I'll update #298.